build/tools/Automation/Python/Constants.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System.Collections.Generic;
+
+namespace Oryx.Microsoft.Automation.Python
+{
+    public class Constants
+    {
+        public const string ConstantsYaml = "constants.yaml";
```

Comment:
Can we reuse the `ConstantsYaml` property defined in the `Automation/Constants.cs` file instead of redefining it here?

Reply:
Yup! Updated here too